include/maths.h

```diff
@@ -33,7 +33,7 @@ inline uint64_t pow_mod_u64(uint64_t b, uint64_t e, uint64_t p) {
   return r;
 }
 
-inline uint64_t inv64_u64(uint64_t p) {
+static inline uint64_t inv64_u64(uint64_t p) {
```

Comment:
Without the static our code doesn't compile at `-O0` because the compiler doesn't inline and without static we don't have a non-inline function definition around 😳